cmd/internal/kube/inject.go

```diff
@@ -41,15 +38,15 @@ var injectCmd = &cobra.Command{
 	Short: "Inject the Postman Insights Agent into a Kubernetes deployment",
 	Long:  "Inject the Postman Insights Agent into a Kubernetes deployment or set of deployments, and output the result to stdout or a file",
 	RunE: func(_ *cobra.Command, args []string) error {
-		if projectNameFlag == "" && postmanCollectionID == "" {
+		if postmanCollectionID == "" {
 			return cmderr.AkitaErr{
-				Err: errors.New("--collection must be specified. Or, if you are an Akita user, use --project instead."),
+				Err: errors.New("--collection must be specified."),
 			}
 		}
```

Comment:
This seems like it is left in an odd state where we still require a collection ID, not a project ID?

Reply:
Yeah, converting this to a project ID was a larger change, and therefore out of scope, since we don't support Kubernetes currently. At least this moves us in the right direction by dropping the conflicting `--project` flag that was used by Akita users to identify an Akita project by name.